clinvar-xml-parser/src/main/java/uk/ac/ebi/eva/clinvar/Application.java

```diff
@@ -0,0 +1,163 @@
+/*
+ *  Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+package uk.ac.ebi.eva.clinvar;
+
+import com.beust.jcommander.JCommander;
+import com.beust.jcommander.Parameter;
+
+import uk.ac.ebi.eva.clinvar.model.ClinvarSet;
+
+import javax.xml.bind.JAXBException;
+import javax.xml.stream.XMLStreamException;
+import java.io.BufferedWriter;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.OutputStreamWriter;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.concurrent.ArrayBlockingQueue;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.zip.GZIPInputStream;
+import java.util.zip.GZIPOutputStream;
+
+public class Application {
+
+    @Parameter(names={"--inputFileName", "-i"}, required = true)
+    private String inputFileName;
+
+    @Parameter(names={"--outputDir", "-o"})
+    private String outputDir = "";
+
+    @Parameter(names={"--clinvarVersion", "-v"}, required = true)
+    private int versionNumber;
+
+    private ExecutorService executor;
+
+    public static void main(String[] args) throws IOException, XMLStreamException, JAXBException {
+        Application app = new Application();
+        JCommander.newBuilder()
+                  .addObject(app)
+                  .build()
+                  .parse(args);
+        app.run();
+    }
+
+    public void run() {
+        ArrayBlockingQueue<String> xmlStringsQueue = new ArrayBlockingQueue<>(1000);
+
```

Comment:
Remove blank line to make it consistent with the next pair of initialized variables?